### ISSUE TITLE
Query results - popups don't work after paging

### ISSUE THREAD

**Issue by julie-sullivan** [intermine/intermine#935]
From @sneuhaus

```
I'm testing out the 1.4 build (merged with our version, which might cause some
issues in itself). It looks like the results tables have changed in a way that is causing 
two types of issues.

A. Mousing over an identifier sometimes causes the table cell to dramatically change 
size (this seems to happen in firefox but not in chrome).

B. When a result table (at least on a details page) is paginated, after using the forward
 and/or go back buttons, frequently (but inconsistently  )the popups displayed when 
mousing over a cell no longer have any data in them. (not browser specific)

I was able to reproduce these on beta.flymine.org (zen gene details page, allele table).
```
#753 covers number A.

**Comment by julie-sullivan** [comment]
From @sneuhaus

```
The cell expansion (second image) seems to be firefox specific.
The loss of detail in the summay (first image) happens in chrome, firefox, opera.
The loss of detail happens after using the go to first/last page or go forward/back 
5 pages buttons.
```

**Comment by julie-sullivan** [comment]
This is fixed on beta.flymine.org. Will want confirmation from @sneuhaus though.

**Comment by sneuhaus** [comment]
On beta.flymnie.org the details page I get after searching for 'zen' and selecting the first result crashes my browser (both firefox and chrome)

**Comment by julie-sullivan** [comment]
Thanks @sneuhaus, Alex is looking at that now. I'll let you know when we are ready for you again. There are a few things to look at, I think.

**Comment by sergiocontrino** [comment]
not sure of the state of this one, but the funny expansion of cells is happening now on firefox (not on chrome).

**Comment by alexkalderimis** [comment]
This should all be resolved as soon as the 2.0 tables are merged
(ie. imminently).

A.

**Comment by julie-sullivan** [comment]
Working in beta flymine. Will ask @sneuhaus to test when the changes are on the beta branch.
